lib/gooder_data/project/report.rb

```diff
@@ -93,6 +94,14 @@ def get_url_report_export(fmt)
         end
       end
 
+      def download_file(url)
+        response = get(url)
+        while response.code == 202 do
```

Comment:
Isn't better to make it more clear using a constrain, for example:

```ruby
ACCEPTED = 202

while response.code == ACCEPTED
  response = get(url)
end
```

Looks more clear for me